### PR TITLE
Update the old install if it exists

### PR DESCRIPTION
**Pull request by QazCetelic**

### CODE REVIEW

**Comment by alex1701c** [comment]
I am not sure about this, because we delete the install_manifext.txt file which references the files from the previous install.

Would it maybe be better to pull the latest changes if we have cloned the project already?

**Comment by QazCetelic** [comment]
Yes, that might work better.

**Comment by alex1701c** [comment]
feel free to create another MR/adjust this one

**Comment by alex1701c** [comment]
@QazCetelic Please adjust this MR if you still think it would be useful to have

File: install.sh [hunk 1]
```diff
@@ -3,7 +3,13 @@
 # Exit if something fails
 set -e
 
-if [[ $(basename "$PWD") !=  "JetBrainsRunner"* ]];then
+# Delete the old install if it exists
```
Comment (alex1701c) [inline install.sh]: The comment is not up to date anymore.

File: install.sh [hunk 2]
```diff
@@ -3,7 +3,13 @@
...
+INSTALL_DIR=~${HOME}/JetBrainsRunner
```
Comment (alex1701c) [inline install.sh]: The script should look in the PWD for the project. In my folder structure, the project is not in $HOME.

File: install.sh [hunk 3]
```diff
@@ -3,7 +3,11 @@
 # Exit if something fails
 set -e
 
-if [[ $(basename "$PWD") !=  "JetBrainsRunner"* ]];then
+if [ -d "$PWD" ]; then
```
Comment (alex1701c) [inline install.sh]: I meant checking "$PWD/JetBrainsRunner", if yes we go inside of it, update the project and proceed to build.

Not sure if I'd update the project when the install.sh script is run from the project dir, maybe leave that as it is for now.

**Pull request merged**